srml/council/src/lib.rs

```diff
@@ -14,138 +14,401 @@
+//!
+//! #### Council Voting
+//!
+//! - `is_vetoed` - Check if a council proposal has been vetoed.
+//! - `will_still_be_councillor_at` - Check each account id of the active council to determine what block number
+//! they will still be active at.
+//! - `is_councillor` - Check if a given account id is a councillor.
+//! - `tally` - The count of the yay and nay votes associated with voting on a council proposal.
+//!
+//!
+//! ### Snippet: Approve all candidates when additional empty seats are available
+//!
+//! This code snippet includes an `approve_all` public function that could be called by a signed account to approve
+//! the eligibility of all candidates when there are empty council seats and when the tally for the next election
+//! occurs at the current or a future block number.
+//!
+//! ```
+//! use srml_support::{decl_module, dispatch::Result};
+//! use system::ensure_signed;
+//! use srml_council::seats::{self as seats};
+//!
+//! pub trait Trait: seats::Trait {}
+//!
+//! decl_module! {
+//!     pub struct Module<T: Trait> for enum Call where origin: T::Origin {
+//!
+//!	        // Approves all candidates.
+//!	        pub fn approve_all(origin) -> Result {
+//!	            let _origin = ensure_signed(origin)?;
+//!
+//!	            // Get the current block number
+//!	            let current_block_number = <system::Module<T>>::block_number();
+//!
+//!	            // Get the number of seats that we want the council to have
+//!	            let desired = <seats::Module<T>>::desired_seats() as usize;
+//!
+//!	            // Get the number of seats occupied by the current council.
+//!	            let occupied = <seats::Module<T>>::active_council().len();
+//!
+//!	            // Get the appropriate block number to schedule the next tally.
+//!	            let maybe_next_tally = <seats::Module<T>>::next_tally();
+//!
+//!	            assert!(desired > occupied, "Unable to approve all candidates when there are no empty seats");
+//!
+//!	            if let Some(next_tally_block_number) = <seats::Module<T>>::next_tally() {
+//!	                if current_block_number <= next_tally_block_number {
+//!	                    assert!(maybe_next_tally.is_some(),
+//!	                        "Unable to approve all candidates when the block number of the next tally has past");
+//!	                }
+//!             }
+//!
+//!             Ok(())
+//!         }
+//!     }
+//! }
+//! # fn main() { }
+//! ```
+//!
+//! ## Related Modules
+//!
+//! - [`System`](../srml_system/index.html)
+//! - [`Support`](../srml_support/index.html)
 
 #![cfg_attr(not(feature = "std"), no_std)]
 
-pub mod voting;
 pub mod motions;
 pub mod seats;
+pub mod voting;
 
-pub use crate::seats::{Trait, Module, RawEvent, Event, VoteIndex};
+pub use crate::seats::{Event, Module, RawEvent, Trait, VoteIndex};
 
 #[cfg(test)]
 mod tests {
-	// These re-exports are here for a reason, edit with care
```

Comment:
Spaces → tabs

Reply:
Thanks i've pushed a commit to restore the indentation now after reviewing the style guide https://wiki.parity.io/Substrate-Style-Guide. Is it ok now?

Reply:
There is a problem with reviewing indentation in GitHub. Not sure if it has to do with my browser dimensions or if other people have it too. I've had correctly-formatted blocks in VS Code and the rendered docs be correct, but when reviewing on GitHub, it's a complete mess. We always want the final html to be correct, so we just have to make sure to download the branch and run `cargo doc` to see how it looks before changing.